awx/ui/client/features/jobs/routes/instanceGroupJobs.route.js

```diff
@@ -63,6 +63,70 @@ export default {
         SearchBasePath: [
             '$stateParams',
             ($stateParams) => `api/v2/instance_groups/${$stateParams.instance_group_id}/jobs`
-        ]
+        ],
     }
 };
+
+const containerGroupJobsRoute = {
+    name: 'instanceGroups.containerGroupJobs',
+    url: '/container_groups/:instance_group_id/jobs',
+    ncyBreadcrumb: {
+        parent: 'instanceGroups.editContainerGroup',
+        label: N_('JOBS')
+    },
+    params: {
+        job_search: {
+            value: {
+                page_size: '10',
+                order_by: '-finished'
+            },
+            dynamic: true
+        }
+    },
+    views: {
+        'instanceGroupsJobsContainer@instanceGroups': {
```

Comment:
I don't think there's a need for the `instances` tab. It's [not on the mockups](https://tower-mockups.testing.ansible.com/mockups/instance_groups/instance-groups-container-edit/) and I think it makes sense that they aren't - there's not a dedicated set of instances to show when container groups are being used.

All we need are the `details` and `jobs` tabs.

Reply:
Oh, I didn't see the mockups in edit.  Thanks for catching that.